test_util/test_upgrade_vpc.py

```diff
@@ -211,7 +211,7 @@ def wait_for_dns(hostname):
         # Skip raising an exception on DNS failure so that other tests can run.
         # DNS failure is being tracked at
         # https://mesosphere.atlassian.net/browse/DCOS-13426.
-        #raise Exception(message)  # noqa
+        # raise Exception(message)  # noqa
```

Comment:
Apparently your python linter doesnt respect ```# noqa``` which prevents any linting errors from that line